x/cosmwasmpool/keeper.go

```diff
@@ -43,3 +42,32 @@ func (k Keeper) GetParams(ctx sdk.Context) (params types.Params) {
 func (k Keeper) SetParams(ctx sdk.Context, params types.Params) {
 	k.paramSpace.SetParamSet(ctx, &params)
 }
+
+// Set the poolmanager keeper.
+func (k *Keeper) SetPoolManagerKeeper(poolmanagerKeeper types.PoolManagerKeeper) {
+	k.poolmanagerKeeper = poolmanagerKeeper
+}
+
+// Set the contract keeper.
+func (k *Keeper) SetContractKeeper(contractKeeper types.ContractKeeper) {
+	k.contractKeeper = contractKeeper
+}
+
+// Set the wasm keeper.
+func (k *Keeper) SetWasmKeeper(wasmKeeper types.WasmKeeper) {
+	k.wasmKeeper = wasmKeeper
+}
+
+// convertToCosmwasmPool converts a poolI to a CosmWasmExtension.
+func (k *Keeper) convertToCosmwasmPool(poolI poolmanagertypes.PoolI) (types.CosmWasmExtension, error) {
```

Comment:
nit: convert makes it sounds like you're modifying the pool, but we're just casting it to the new type. Maybe rename to something like `asCosmwasmPool()`?

Reply:
renamed as well as relevant helpers in other modules